airflow/ui/src/components/TriggerDag/TriggerDAGForm.tsx

```diff
@@ -204,6 +204,17 @@ const TriggerDAGForm: React.FC<TriggerDAGFormProps> = ({
                   </Box>
                 )}
               />
+
+              <Text fontSize="md" mb={2} mt={6}>
+                Notes
```

Comment:
Can we make clear that this are not just "Any Notes" but "Dag Run Notes"?

(If I correctly assume from view of legacy UI this refers to the notes as below?)
![image](https://github.com/user-attachments/assets/45d68394-d36c-461c-9ae1-acfbaf91cdfd)

```suggestion
                Dag Run Notes
```